app/screens/index.ts

```diff
@@ -1,4 +1,5 @@
+import InitiativeScreen from './initiative-screen'
```

Comment:
After rerunning the latest of Hydrogen (the approved PR on master)
`import InitiativeScreen from './Initiative'`